### PR TITLE
 virtualbox: security update to 6.1.30; #3664 

### PR DESCRIPTION
**Pull request by WhiredPlanck**
Topic Description
-----------------

Security update `virtualbox` to 6.1.30

Package(s) Affected
-------------------

`virtualbox`

Security Update?
----------------

Yes - Issue Number: #3664 

Test Build(s) Done
------------------

**Primary Architectures**

- [x] AMD64 `amd64`   
    

Update(s) Uploaded to Stable
----------------------------

**Primary Architectures**

- [ ] AMD64 `amd64`

### CODE REVIEW

**Comment by WhiredPlanck** [comment]
LGTM on AMD64

图片

File: extra-emulation/virtualbox/01-host/build [hunk 1]
```diff
@@ -1,90 +1,109 @@
+# Adapted from Arch Linux
 VER=$PKGVER
 
-./configure \
```
Comment (MingcongBai) [inline extra-emulation/virtualbox/01-host/build:3]: Missing abinfo.

File: extra-emulation/virtualbox/01-host/build [hunk 2]
```diff
@@ -1,90 +1,109 @@
...
+"$SRCDIR"/configure \
     --disable-docs \
     --enable-webservice \
     --enable-vde \
     --enable-vnc \
     --disable-kmods \
     --with-makeself=/usr/bin/echo
 
-source ./env.sh
+source "$SRCDIR"/env.sh
```
Comment (MingcongBai) [inline extra-emulation/virtualbox/01-host/build:13]: Missing abinfo.

File: extra-emulation/virtualbox/01-host/build [hunk 3]
```diff
@@ -1,90 +1,109 @@
...
-cp -r src "$PKGDIR"/usr/src/vboxhost-$VER
+cp -rv "$OUTPUTDIR"/src "$PKGDIR"/usr/src/vboxhost-$VER
 
-pushd additions
+abinfo "Installing guest utils ..."
+pushd "$OUTPUTDIR"/additions
 
-# Forward port DKMS
-cat > "$PKGDIR"/usr/src/vboxhost-TEMPVER/dkms.conf << EOF
+abinfo "Forward port DKMS"
```
Comment (MingcongBai) [inline extra-emulation/virtualbox/01-host/build]: What?

File: extra-emulation/virtualbox/01-host/build [hunk 4]
```diff
@@ -157,6 +176,5 @@ fi
 exit 1
 EOF
 
-chmod +x "$PKGDIR"/usr/src/vboxhost-TEMPVER/do_dkms
-chmod 0755 "$PKGDIR"/usr/lib/virtualbox/VirtualBox
-
+chmod -v +x "$PKGDIR"/usr/src/vboxhost-$PKGVER/do_dkms
```
Comment (MingcongBai) [inline extra-emulation/virtualbox/01-host/build:179]: Missing abinfo.

File: extra-emulation/virtualbox/02-guest/build [hunk 5]
```diff
@@ -1,24 +1,25 @@
 VER=$PKGVER
-source ./env.sh
```
Comment (MingcongBai) [inline extra-emulation/virtualbox/02-guest/build:2]: Missing abinfo throughout.

**Review by MingcongBai — CHANGES_REQUESTED** [review]

**Comment by WhiredPlanck** [comment]
And ... (remind myself):
- Don’t use abbreviations for topic names.
- One should modify `postinst`, `prerm` etc. via `sed` command in prepare to use the value of a variable (like `$PKGVER`).

**Review by MingcongBai — DISMISSED** [review]

**Review by MingcongBai — APPROVED** [review]

**Pull request merged**